cdptools/tests/sr_models/test_webvtt_sr_model.py

```diff
@@ -34,27 +29,82 @@ def test_webvtt_sr_model_request_caption_content(example_webvtt_sr_model):
         example_webvtt_sr_model._request_caption_content("invalid-caption-uri")
 
 
-# Check whether timestamped sentences of fake_caption equals fake_timestamped_sentences
-def test_webvtt_sr_model_get_sentences(example_webvtt_sr_model, fake_caption, fake_timestamped_sentences):
+@pytest.mark.parametrize("captions, expected", [
+    (
+        [
+            Caption(text="&gt;&gt; Start of Dialog 1."),
+            Caption(text="End of Dialog 1."),
+            Caption(text="&gt;&gt; [ APPLAUSE ] "),
+            Caption(text="&gt;&gt; Dialog 2.")
+        ],
+        [2, 1, 1]
```

Comment:
Can you change the expectation to be the expected strings instead of just sentence count per speaker. It is a harsher check but I would assume easier to read and "safer".

Reply:
Yes,  will do.

Reply:
Yea that would be great!

Reply:
New commits has the changes.